test/specs/log.js

```diff
@@ -15,18 +15,24 @@ describe('log', () => {
 
   describe('removing', () => {
     it('should return the stream', () => {
-      expect(stream().log().offLog()).to.be.observable.stream()
+      expect(
+        stream()
+          .log()
+          .offLog()
+      ).to.be.observable.stream()
```

Comment:
There are a couple of changes like this that not only seem worse than what we had before but are reformatting lines that don't exceed the 120 char `printWidth` config. Are we sure this is configured properly?

Reply:
It's possible this was a prettier opinion change tho.

Reply:
Yeah, sorry, testing on the prettier playground suggests this is intended.